<?xml version="1.0" ?>
<sdf version="1.5">
  <model name="cat_black">
    <link name="link">
      <pose>0 0 0 0 0 0</pose>
      <collision name="collision">
        <geometry>
          <mesh>
            <uri>model://cat_black/meshes/Cat_v1_l3.obj</uri>
            <scale>0.01 0.01 0.01</scale>
          </mesh>
        </geometry>
        <pose>-0.09 0 0 0 0 1.570796327</pose>
      </collision>
      <visual name="visual">
        <geometry>
          <mesh>
            <uri>model://cat_black/meshes/Cat_v1_l3.obj</uri>
            <scale>0.01 0.01 0.01</scale>
          </mesh>
        </geometry>
        <pose>-0.09 0 0 0 0 1.570796327</pose>
      </visual>
      <inertial>
        <inertia>
          <ixx>0.03333</ixx>
          <ixy>0</ixy>
          <ixz>0</ixz>
          <iyy>0.08333</iyy>
          <iyz>0</iyz>
          <izz>0.05667</izz>
        </inertia>
        <mass>4</mass>
        <pose>0 0 0.15 0 0 0</pose>
      </inertial>
    </link>
  </model>
</sdf>
